test/test_manifest.json

```diff
@@ -5899,5 +5899,12 @@
        "rounds": 1,
        "lastPage": 1,
        "type": "eq"
+    },
+    {  "id": "bug1722036",
+       "file": "pdfs/bug1722036.pdf",
+       "md5": "bbc7f62ca85609c97069bd3ce2658288",
+       "rounds": 1,
+        "link": true,
```

Comment:
Nit: Too much indentation here.

Reply:
Also, isn't this test-case passing already without the patch!?

I think you want to add `"forms": true` as well here, right?